src/Microsoft.Health.Dicom.Core/Features/CustomTag/GetCustomTagHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Dicom.Core.Features.Validation;
+using Microsoft.Health.Dicom.Core.Messages.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    public class GetCustomTagHandler : IRequestHandler<GetCustomTagRequest, GetCustomTagResponse>
+    {
+        private readonly IGetCustomTagsService _getCustomTagsService;
+
+        public GetCustomTagHandler(IGetCustomTagsService getCustomTagsService)
+        {
+            EnsureArg.IsNotNull(getCustomTagsService, nameof(getCustomTagsService));
+            _getCustomTagsService = getCustomTagsService;
+        }
+
+        public async Task<GetCustomTagResponse> Handle(GetCustomTagRequest request, CancellationToken cancellationToken)
+        {
+            TagPathValidator.Validate(request.CustomTagPath);
```

Comment:
I checked in DicomTagParser (https://github.com/microsoft/dicom-server/blob/master/src/Microsoft.Health.Dicom.Core/Features/Common/DicomTagParser.cs) in previous PR for parsing tag path.  maybe we can reuse that?

Reply:
I'll take a look!

---
In reply to: [558528870](https://github.com/microsoft/dicom-server/pull/501#discussion_r558528870) [](ancestors = 558528870)